docs/docs/hydroflow_plus/consistency.mdx

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 3
+---
+
+# Consistency and Safety
+A key feature of Hydroflow+ is its integration with the Rust type system to highlight possible sources of inconsistent distributed behavior due to sources of non-determinism such as batching, timeouts, and message reordering. In this section, we'll walk through the consistency guarantees in Hydroflow+ and how to use the **`unsafe`** keyword as an escape hatch when introducing sources of non-determinism.
+
+:::note
+
+Our consistency and safety model is based on the POPL'25 paper [Flo: A Semantic Foundation for Progressive Stream Processing](https://arxiv.org/abs/2411.08274), which covers the formal details and proofs underlying this system.
+
+:::
+
+## Determinism and Eventual Consistency
+Much existing literature in distributed systems focuses on consistency levels such as "eventual consistency" which typically correspond to guarantees when reading the state of an object (or set of objects) at a _specific point_ in time. Hydroflow+ does not use such a consistency model internally, since it uses a streaming model where values are being continuously updated _over time_.
```

Comment:
Seems like apples to oranges, eventual consistency really is just for replicated data? Determinism seems like a different thing

Reply:
Maybe would be better to have this paragraph after/tied to "eventual determinism," seems analogous